tests/mask.py

```diff
@@ -99,10 +99,10 @@ def setUp(self):
         except Exception:
             self.maskFile = None
         else:
-            if True:
-                self.maskFile = os.path.join(dataDir, "small_MI.fits")
-            else:
-                self.maskFile = os.path.join(dataDir, "871034p_1_MI.fits")
+            self.maskFile = os.path.join(dataDir, "small_MI.fits")
+            # Above mask looks like this, possibly left-shifted by some number of mask planes.
```

Comment:
I'll not change to not use proper skipping, but I can put in a "#skipif" note or somesuch, if there's a common way to refer to them.